### PR TITLE
EVMJIT: stack improvements

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: evmjit/libevmjit/BasicBlock.cpp [hunk 1]
```diff
@@ -107,42 +119,91 @@ void LocalStack::set(size_t _index, llvm::Value* _word)
...
+
+	m_builder.SetInsertPoint(outOfStackBB);
+	auto longjmp = llvm::Intrinsic::getDeclaration(getModule(), llvm::Intrinsic::eh_sjlj_longjmp);
+	m_builder.CreateCall(longjmp, {jmpBuf});
+	m_builder.CreateUnreachable();
+
+	return func;
+}
+
+
```
Comment (CJentzsch) [inline evmjit/libevmjit/BasicBlock.cpp:206]: two blank lines in a row

File: evmjit/libevmjit/Compiler.cpp [hunk 2]
```diff
@@ -136,26 +152,30 @@ std::unique_ptr<llvm::Module> Compiler::compile(code_iterator _begin, code_itera
...
+	auto stopBB = llvm::BasicBlock::Create(m_mainFunc->getContext(), "Stop", m_mainFunc);
 
-	m_stopBB = llvm::BasicBlock::Create(m_mainFunc->getContext(), "Stop", m_mainFunc);
-	m_abortBB = llvm::BasicBlock::Create(m_mainFunc->getContext(), "Abort", m_mainFunc);
 	m_jumpTableBB = llvm::BasicBlock::Create(m_mainFunc->getContext(), "JumpTable", m_mainFunc);
-	m_builder.SetInsertPoint(m_jumpTableBB);
+
+	auto abortBB = llvm::BasicBlock::Create(m_mainFunc->getContext(), "Abort", m_mainFunc);
+
+
```
Comment (CJentzsch) [inline evmjit/libevmjit/Compiler.cpp]: two blank lines in a row

File: evmjit/libevmjit/BasicBlock.cpp (hunk 1 above)
Comment (chfast) [inline evmjit/libevmjit/BasicBlock.cpp:206]: I separate functions with 2 blank lines.

**Comment by CJentzsch** [comment]
Did you run all the tests for it?

**Comment by chfast** [comment]
StateTests + VMTests.

I will check again on Windows. Let me also fix one of the double lines issues.

**Comment by CJentzsch** [comment]
I am currently running them on linux

**Comment by chfast** [comment]
No errors on Windows. Nice. I haven't run Windows tests for a long time.

**Comment by CJentzsch** [comment]
No errors on linux

**Comment by arkpar** [comment]
looksgood

**Pull request merged**